modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties

```
@@ -328,6 +328,7 @@ category.shipping-tax=Shipping Tax
 category.site_administration.commerce=Commerce
 category.users=Users
 category.wish-lists=Wish Lists
+cdn-url=Cdn Url
```

We should have them capitalized since they are abbreviations.

```suggestion
cdn-url=CDN URL
```